src/bin/ion/commands/from/json.rs

```diff
@@ -16,11 +16,20 @@ impl IonCliCommand for FromJsonCommand {
     }
 
     fn is_stable(&self) -> bool {
+        false // TODO: Should this be true?
```

Comment:
We've debated over whether or not there should be upconversion of e.g. timestamps before... but it seems like that's not something you'd want as the default behavior. Ideally it seems like you'd want the default to be a stable, lossless transformation. 

I guess what I'm saying is that I don't see any reason this couldn't be stable.